storm.yaml

```diff
@@ -4,27 +4,36 @@
 mesos.master.url: "zk://localhost:2181/mesos"
 storm.zookeeper.servers:
     - "localhost"
-nimbus.host: "localhost"
 # -----------------------------------------------------------
 
 
-### If this is not false then you need to add about 128m of mem usage to executor to account for the logviewer
-#supervisor.autostart.logviewer: false
-topology.mesos.executor.mem.mb: 1128  ## default is 1000
+# Worker resources
+topology.mesos.worker.cpu: 1.0
+### Worker heap with 20% overhead
+topology.mesos.executor.mem.mb: 1250
+worker.childopts: "-Xmx1000m"
 
-# The default behavior is to launch the logvierwer unless autostart is false
-logviewer.port: 8000
+# Supervisor resources
+topology.mesos.executor.cpu: 0.1
+topology.mesos.executor.mem.mb: 500 # Supervisor memory, with 20% overhead
+supervisor.childopts: "-Xmx256m"
+
+# The default behavior is to launch the logviewer unless autostart is false.
+# If you enable the logviewer, you'll need to add memory overhead to the
+# executor for the logviewer.
 logviewer.childopts: "-Xmx128m"
 logviewer.cleanup.age.mins: 10080
 logviewer.appender.name: "A1"
-
-
-# You should not need to change anything below this line
-#--------------------------------------------------------
+supervisor.autostart.logviewer: true
```

Comment:
What I meant was that you could not have 2 supervisors from the same nimbus, per IP (or host machine)--since they'd both try to bind a logviewer to the same port.